src/mono/sample/wasm/browser-advanced/index.html

```diff
@@ -11,8 +11,6 @@
   <script type='module' src="./dotnet.js"></script>
   <link rel="preload" href="./mono-config.json" as="fetch" crossorigin="anonymous">
   <link rel="prefetch" href="./dotnet.wasm" as="fetch" crossorigin="anonymous">
-  <link rel="prefetch" href="./icudt.dat" as="fetch" crossorigin="anonymous">
```

Comment:
do we know how serious the negative impact of this is on first load? it's valuable on later loads, of course. it may be worth evaluating whether prefetches should be kept, since on later loads the assets are probably already in the browser cache so the prefetch won't do anything bad

Reply:
It would keep it alive in the cache. Is that bad enough ?

Reply:
will revert this